seedScript.js

```diff
@@ -68,3 +68,9 @@ const seed = async () => {
 };
 
 seed();
+module.exports = {
+  insertProducts,
+  createNewProducts,
+  gtinStr,
+  randNum,
+};
```

Comment:
Does this work? Since when you import this, it will seed the database before you can use these functions? and the functions themselves don't clear out the database? unless you are using these functions in a way I'm not currently seeing?